src/components/AssignmentRevokeModal/index.jsx

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
```

Comment:
as per my understanding we are revoking an assignment so shouldn't we name it `AssignmentRevokeModal` or `CodeAssignmentRevokeModal`?

Reply:
@muhammad-ammar Hmm... to me, I feel like the components should be named based on the action we're doing for the code(s). For assignment, it's named `CodeAssignmentModal` because we're assigning a code. I was thinking for revoke/remind, we would follow a similar convention where we name it based on the action, e.g. `CodeRevokeModal`.

If we did use `AssignmentRevokeModal` or `CodeAssignmentRevokeModal`, I feel like it might be a bit confusing when we already have `CodeAssignmentModal`.

Thoughts?

Reply:
OK. In that case it make sense.